packages/accordion/__tests__/Accordion.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { Accordion, AccordionItem } from '@littlespoon/accordion/src/Accordion'
+import { AccordionDetails } from '@littlespoon/accordion/src/AccordionContent'
```

Comment:
Can we rename this file to match the name of the component?  `AccordionContent` => `AccordionDetails`